backend/apps/companies/tests/test_service_transactions.py

```
@@ -0,0 +1,320 @@
+
+    def test_company_create_rollback_on_duplicate_error(self):
+        """
+        Test that company_create rolls back when duplicate is detected.
+
+        Scenario:
+        1. Create first company
+        2. Attempt to create duplicate
+        3. Verify only first company exists (rollback succeeded)
+        """
+        # Create first company
+        company1 = CompanyService.company_create(
```

Ruff (0.14.2)

Local variable `company1` is assigned to but never used

Remove assignment to unused variable `company1`

(F841)